src/posts/official-ppa-20230206.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: "/news/official-ppa-20230206"
+date: "2023-02-06"
+title: "Announcing Apptainer Official PPA for Ubuntu"
+---
+
+We are pleased and excited to announce that Apptainer official PPA for Ubuntu 
+is now available!
+
+Rocky Linux users have long benefited from the availability of Apptainer on
```

Comment:
For the apptainer announcement let's not bias it to Rocky.  Say "Users of RHEL-derived operating systems".

Reply:
Thank you, I'm preparing to update PR right now.